core/src/main/java/com/exadel/aem/toolkit/bundle/lists/datasource/ChildResourcesDatasource.java

```diff
@@ -50,19 +49,19 @@
 import com.adobe.granite.ui.components.ds.EmptyDataSource;
 
 /**
- * Servlet that implements {@code datasource} pattern for populating a Custom Lists Console
- * with all child pages under the current root path, which are either custom lists themselves,
+ * Servlet that implements {@code datasource} pattern for populating a Lists Console
```

Comment:
I would put it as `Provides the collection of AEM resources that either represent AAT Lists or serve as folders for AAT lists to be displayed in the AAT Lists console`

Reply:
Reformulated